operator/deploy/olm-catalog/0.1.0/automationbrokeroperator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -1,10 +1,11 @@
+---
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   name: automationbrokeroperator.v0.1.0
   namespace: placeholder
   annotations:
```

Comment:
Read through https://docs.google.com/document/d/10SJ_k7efVMnNxv49AT-jTdvnKF_O0R1CFdT_-KOW1Qg/edit and I want to state what I believe to be the relevant configuration (primarily CSV) for the ansible-service-broker(-operator) to be installed in such a way that the Operator is installed in `openshift-operators` (which to my understanding is the preferred method for 4.0) watching a single namespace `openshift-ansible-service-broker`. This way if I'm wrong it should be easy to point out what I'm not understanding:

1) Annotations on CSV:

```
metadata:
  name: automationbrokeroperator.v0.1.0
  namespace: placeholder
  annotations:
    olm.targetNamespaces: openshift-ansible-service-broker
    olm.operatorNamespace: openshift-operators
    olm.operatorGroup: openshift-production
```

2) Install Modes:

```
spec:
  installModes:
  - type: OwnNamespace
    supported: false
  - type: SingleNamespace
    supported: true
  - type: MultiNamespace
    supported: false
  - type: AllNamespaces
    supported: false
```

3) Setting WATCH_NAMESPACE environment variable for operator built with operator-sdk:

```
env:
- name: WATCH_NAMESPACE
  valueFrom:
      fieldRef:
          fieldPath: metadata.annotations['olm.targetNamespaces']
```